tests/printer/expr/record.res

```diff
@@ -28,3 +28,13 @@ let user = {name: {
 }}
 // braced + constrained expr
 let user = {name: {(ceo.name: string)}}
+
+
+// Punning
+let r = {a} // actually not a record, just an expression in braces
+let r = {a, b}
```

Comment:
Can you add an extra test case with all kind of different comments on punned key-values?

Reply:
Added some, looks good to me.